.github/workflows/boefjes_container_image.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         id: buildx
 
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
+        uses: docker/login-action@v2.2.0
```

Comment:
This will only cause work to bump version numbers. Given that this hasn't caused us any problems in the past and is the normal way to specify actions as far as I know, I think we should just keep `@v2`. I think dependabot only changed it because in another place we specified a specific version.